poi_fr.md

```diff
@@ -0,0 +1,211 @@
+`clinic` | `amenity` | 
+`hospital` | `amenity` | 
+`place_of_worship` | `amenity` | Établissement religieux
+`post_office` | `amenity` | 
+`car_sharing` | `amenity` | Zone de parking pour les voitures "partagées" de type AutoLib (équivalent des VLS pour les voitures)
+`carpooling` | `amenity` | Aire de covoiturage
+`bicycle_rental` | `amenity` | Location de vélos (seulement les grands réseaux de type Vélib'). Permet de définir les stations de VLS utilisés dans navitia.
+`bicycle_parking` | `amenity` | Parking à vélos (pas utilisé pour le rabattement vélo de Navitia)
+`parking` | `amenity` | Aires de stationnement (P+R ou non). Utilisé pour définir les points de rabattement voiture dans une recherche d'itinéraire.
+`public_building` | `amenity` | Tag déconseillé par OSM mais encore beaucoup utilisé.
+`police` | `amenity` | 
+`townhall` | `amenity` | Mairies
+`community_centre` | `amenity` | Salles de spectacle, salles des fêtes ou locaux d'associations
+`embassy` | `amenity` | ambassades (si présentes)
+`park` | `leisure` | Jardins et parcs
+`stadium` | `leisure` | 
+`museum` | `tourism` | 
+`zoo` | `tourism` | 
+`aerodrome` | `aeroway` | 
+`terminal` | `aeroway` | 
+
+
+# Liste des catégories complémentaires possibles
+
+OSM value | Source OSM | Commentaire
+--- | --- | ---
+`social_facility` | `amenity` | CAF et autres aides sociales qui ne sont pas déjà dans `public_building`
+`employment_agency` | `amenity` | Agences pour l'emploi qui ne sont pas déjà dans `public_building`
+`pitch` | `leisure` | [petit terrain de sport](http://wiki.openstreetmap.org/wiki/FR:Tag:leisure%3Dpitch)
+`garden` | `leisure` | 
+`camp_site` | `tourism` | aire de camping
+`swimming_pool` | `tourism` | piscines
+`water_park` | `tourism` | bases nautiques
+`supermarket` | `shop` | [supermarchés](http://wiki.openstreetmap.org/wiki/FR:Tag:shop=supermarket)
+`mall` | `shop` | centre commerciaux
+
+
+# Cas particulier de l'alimentation par un PBF OSM
+
+Les catégories extraites ne sont alors pas paramétrables et correspondent aux
+catégories suivantes :
+
+```
+data.poi_types =
+ {
+    {"amenity:college" , new ed::types::PoiType(0,  "école")},
+    {"amenity:university" , new ed::types::PoiType(1, "université")},
+    {"amenity:theatre" , new ed::types::PoiType(2, "théâtre")},
+    {"hospital" , new ed::types::PoiType(3, "hôpital")},
+    {"amenity:post_office" , new ed::types::PoiType(4, "bureau de poste")},
+    {"amenity:bicycle_rental" , new ed::types::PoiType(5, "station vls")},
+    {"amenity:bicycle_parking" , new ed::types::PoiType(6, "Parking vélo")},
+    {"amenity:parking" , new ed::types::PoiType(7, "Parking")},
+    {"amenity:police" , new ed::types::PoiType(8, "Police, Gendarmerie")},
+    {"amenity:townhall" , new ed::types::PoiType(9, "Mairie")},
+    {"leisure:garden" , new ed::types::PoiType(10, "Jardin")},
+    {"leisure:park" , new ed::types::PoiType(11, "Zone Parc. Zone verte ouverte, pour déambuler. habituellement municipale")}
+};
+```
```

Comment:
I've removed a last sentence here that makes a link with another internal documentation. It's possible that we could replace this link with something that points to [that](https://doc.navitia.io/#street-network-objects) but I'm not sure.

Reply:
Outdated since this whole part has been removed.